Gemfile

```diff
@@ -13,6 +13,7 @@ gem "puma"
 gem "rails", "4.2.0"
 gem "sass-rails", "~> 5.0"
 gem "uglifier", ">= 1.3.0"
+gem "airbrake"
```

Comment:
I like to keep the Gemfile in alphabetical order. Easier to find things that way.